scan-command/src/main/java/io/ballerina/scan/utils/ScanUtils.java

```diff
@@ -550,4 +551,38 @@ private static Optional<String> loadRemoteJAR(Path targetDir, String fileName, U
             return Optional.empty();
         }
     }
+
+    /**
+     * Prints the {@link List<Rule>} of static analysis rules to the console.
+     *
+     * @param rules        the list of static analysis rules
+     * @param outputStream the print stream
+     */
+    public static void printRulesToConsole(List<Rule> rules, PrintStream outputStream) {
+        String ruleIDColumn = "RuleID";
+        String ruleSeverityColumn = "Rule Severity";
+        String ruleDescriptionColumn = "Rule Description";
+
+        int maxRuleIDLength = ruleIDColumn.length();
+        int maxSeverityLength = ruleSeverityColumn.length();
+        int maxDescriptionLength = ruleDescriptionColumn.length();
+
+        for (Rule rule : rules) {
+            maxRuleIDLength = Math.max(maxRuleIDLength, rule.id().length());
+            maxSeverityLength = Math.max(maxSeverityLength, rule.kind().toString().length());
+            maxDescriptionLength = Math.max(maxDescriptionLength, rule.description().length());
+        }
+
+        String format = "\t%-" + maxRuleIDLength + "s | %-" + maxSeverityLength + "s | %-" + maxDescriptionLength
+                + "s%n";
+
+        outputStream.printf(format, ruleIDColumn, ruleSeverityColumn, ruleDescriptionColumn);
+        outputStream.printf("\t" + "-".repeat(maxRuleIDLength + 1) + "--" +
+                "-".repeat(maxSeverityLength + 1) + "--" + "-".repeat(maxDescriptionLength + 1) + "%n");
+
+        for (Rule rule : rules) {
+            String formattedLine = String.format(format, rule.id(), rule.kind().toString(), rule.description());
+            outputStream.println(formattedLine.replaceAll("\\s+$", ""));
```

Comment:
Are you trying to remove trailing white spaces here? If so I think it is more clear to just use `formattedLine.stripTrailing()`